libs/ledger-live-common/src/families/ethereum/signOperation.ts

```diff
@@ -97,7 +98,11 @@ export const signOperation = ({
                 }
 
                 const rawData = tx.raw();
-                rawData[6] = Buffer.from([common.chainIdBN().toNumber()]);
+                rawData[6] = Buffer.from(
+                  common.chainIdBN().toString("hex"),
+                  "hex"
+                );
```

Comment:
Could this be an opportunity to cover this with tests?

Reply:
Created a ticket for this, will do